depends/patches/qt/qttools_skip_dependencies.patch

```diff
@@ -0,0 +1,36 @@
+QtTools: Skip unnecessary dependencies:
+
+1. The LLVM installatiion.
+2. Build only required tools.
```

Comment:
In 9d3be36f758095167ddd762400fda1a3ea76e8f9:

> 2. Build only required tools.

How extensive should this be? Looking at the bin/libexec depends dirs after a build of this branch, we seem to end up with a lot of tooling/scipts/binaries that we are not using. For example:
```bash
qt-cmake-private
qt-internal-configure-examples
qt-internal-configure-tests
qt-testrunner.py
qtpaths
qtpaths6
qvkgen
sanitizer-testrunner.py
syncqt
tracegen
tracepointgen
```

Not sure if that should be getting cleaned up here, or in a post-install phase, but ideally we can skip compiling stuff we never use.

Reply:
I've [added](https://github.com/bitcoin/bitcoin/pull/30997#issuecomment-2760340414) a new patch that addresses you comment for trivial cases. We can address other cases, where tools are more deeply integrated in the Qt's build system, in follow-ups.